python/ray/serve/api.py

```diff
@@ -236,14 +237,30 @@ async def __init__(self, *args, **kwargs):
                 ):
                     await self._serve_asgi_lifespan.startup()
 
-            async def __call__(self, request: Request):
-                sender = ASGIHTTPSender()
+            async def __call__(
```

Comment:
Could you provide some insights why we need to have this change? difficult to understand this scenario :)

Reply:
I can update the code with comments too, but this is to continue the existing codepath where we expect a `Response` to be returned (as was written before) as well as the new path where we pass in a `Send` interface for fastapi to call directly